File: app/src/main/java/com/tongfu/mytestapp/recyclerview/base/VH.java
package com.tongfu.mytestapp.recyclerview.base;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.tongfu.mytestapp.R;

import butterknife.BindView;
import butterknife.ButterKnife;

final class VH extends RecyclerView.ViewHolder  {
    @BindView(R.id.tv_name)
    public TextView tvName ;

    public VH(@NonNull View itemView) {
        super(itemView);
        ButterKnife.bind(this , itemView);
    }

//    @Override
//    public String toString() {
//        return super.toString();
//    }
}

File: app/src/main/java/com/tongfu/mytestapp/recyclerview/swipemenu/VH.kt
package com.tongfu.mytestapp.recyclerview.swipemenu

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import butterknife.BindView
import butterknife.ButterKnife
import com.tongfu.mytestapp.R

internal class VH(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @BindView(R.id.tv_name_a)
    lateinit var tv :TextView
    init{
        ButterKnife.bind(this , itemView )
    }
}